sw/device/lib/drivers/dma/dma.c

```diff
@@ -24,3 +24,15 @@ void dma_set_cnt_start(const dma_t *dma, uint32_t copy_size) {
 int32_t dma_get_done(const dma_t *dma) {
   return mmio_region_read32(dma->base_addr, (ptrdiff_t)(DMA_DONE_REG_OFFSET));
 }
+
+void dma_set_read_ptr_inc(const dma_t *dma, uint32_t read_ptr_inc){
+  mmio_region_write32(dma->base_addr, (ptrdiff_t)(DMA_SRC_PTR_INC_REG_OFFSET), read_ptr_inc);
+}
+
+void dma_set_write_ptr_inc(const dma_t *dma, uint32_t write_ptr_inc){
+  mmio_region_write32(dma->base_addr, (ptrdiff_t)(DMA_DST_PTR_INC_REG_OFFSET), write_ptr_inc);
+}
+
+void dma_set_spi_mode(const dma_t *dma, uint32_t spi_mode){
+  mmio_region_write32(dma->base_addr, (ptrdiff_t)(DMA_SPI_MODE_REG_OFFSET), spi_mode);
+}
```

Comment:
Always keep an empty line at the end of any file to avoid the wrath of terminal users